components/player/VideoPlayer.tsx

```diff
@@ -1,15 +1,57 @@
-import React from 'react';
-import { View } from 'react-native';
+/* eslint-disable react/no-unstable-nested-components */
+import { useTheme } from '@react-navigation/native';
+import { ResizeMode, Video } from 'expo-av';
+import { Image } from 'expo-image';
+import React, { useRef } from 'react';
+import { Dimensions, StyleSheet, View } from 'react-native';
+import { VIDEO_URL } from '../../config/constants';
 
-// TODO: Add VideoPlayer component
-/**
- * -  Implement a video player card component
- * - Hint: use `expo-av` to implement the video player with hidden controls
- * - consider adding fullscreen player
- */
+const { width } = Dimensions.get('window');
 
 const VideoPlayer = () => {
-  return <View />;
+  const { colors } = useTheme();
+
+  const video = useRef(null);
+
+  return (
+    <View style={[styles.container, { backgroundColor: colors.card }]} testID="video-player">
+      <Video
+        testID="video-component"
+        ref={video}
+        style={styles.video}
+        source={{
+          uri: VIDEO_URL,
+        }}
+        useNativeControls
+        resizeMode={ResizeMode.CONTAIN}
+        isLooping
+        isMuted
+        shouldPlay
+        PosterComponent={() => (
+          <Image source={require('../../assets/images/banner.png')} style={styles.image} contentFit="contain" />
+        )}
```

Comment:
I tested this on slow 3G and the poster is not loaded. I believe we need to set the `usePoster` prop to `true` for the image to be displayed.

![image](https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/792405ee-cf01-4147-af93-08aa8469f723)

Also `contentFit="contain"` does not fill up the poster to the container, maybe `none` works better?

<img width="570" alt="image" src="https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/c03ac9fd-6c51-410d-ba2e-00113c689697">

Reply:
`usePoster` prop fixes this 🤘🏽

Reply:
also `contentFit="none"` works with `contentPosition="center"` 👌🏽